app/views/layouts/application.html.erb

```diff
@@ -7,7 +7,7 @@
     <%= csp_meta_tag %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
-    <%= javascript_importmap_tags %>
+    <%= javascript_pack_tag 'hello_react' %>
```

Comment:
- [ ] The partial for the react file should be in a separate folder, and the javascript tag should remain as it was at the creation of the app.

hint : <%= react_component("App") %>

Reply:
Hi @dasileker

If I include the `<%= javascript_importmap_tags %>` there's double rendering and the greeting appears twice.
Any ideas on how to resolve that?
your `<%= react_component("App") %>` code doesn't work.